dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/template/AlertTemplateFactory.java

```diff
@@ -30,25 +27,13 @@
 
     private static final Logger logger = LoggerFactory.getLogger(AlertTemplateFactory.class);
 
-    private static final String alertTemplate = PropertyUtils.getString(Constants.ALERT_TEMPLATE);
-
     private AlertTemplateFactory(){}
 
     /**
      * get a template from alert.properties conf file
      * @return a template, default is DefaultHTMLTemplate
      */
     public static AlertTemplate getMessageTemplate() {
-
-        if(StringUtils.isEmpty(alertTemplate)){
-            return new DefaultHTMLTemplate();
-        }
-
-        switch (alertTemplate){
```

Comment:
why delete these?
We can add like 'simpleTextTemplate' implementations in the future, and config it simply.

Reply:
I think we can add simpleTextTemplate again in the future, not complicated